chcemvediet/apps/inforequests/templates/admin/inforequests/action/delete_selected_confirmation.html

```diff
@@ -0,0 +1,34 @@
+{% extends "admin/delete_selected_confirmation.html" %}
+{% load amend before after set_attributes from poleno.amend %}
+
+{% block content %}
+  {% amend %}
+    {{ block.super }}
+    {% after path="./ul[last()]" %}
+      {% if outbound %}
+        <p>Outbound messages will be deleted:</p>
+        <ul>{{ outbound|unordered_list }}</ul>
+      {% endif %}
+      {% if inbound %}
+        <p>Inbound messages will be marked undecided:</p>
+        <ul>{{ inbound|unordered_list }}</ul>
+      {% endif %}
+    {% endafter %}
+    {% before path=".//form" %}
+      {% if warnings %}
+        <ul class="messagelist">
```

Comment:
@martinmacko47 Podobne by som to spravil aj na vymazavani samostatneho Action modelu.

Reply:
@viliambalaz jj, dobry napad. Len to pls premenuj z `warnings` na `delete_warnings`, nech je to symetricke s `delete_constraints`.

Navyse logiku vytvarajucu jednotlive warningy by som vysunul do separatnej metody `delete_warnings` analogicky ako mame metodu `delete_constraints`. Nech sa ta metoda da zavolat pri bulk delete aj pri klasickom delete. Aby logiku, kedy sa ma vypisat konkretny warning sme mali v kode iba raz. A aby sa v buducnosti pri tej metode dalo vyuzit dedenie.

Reply:
:ok: Trocha som to este prerobil, vyhodil som podmienku `if not obj_constraints:`, predpokladam, ze ak warningy budu existovat, tak nebudu ziadne obmedzenia pre vymazanie.

Reply:
beriem späť, tento predpoklad bol chybny, tak podmienku som vratil. Je si myslim zavadzajuce, ak vypisujeme warning no objekt, ktory nemoze admin vymazat.

Reply:
Neviem. IMHO kludne mozeme vypisat aj constrainy aj warningy pre jeden a ten isty object. Teda, ze ho nejde zmazat pretoze X, ale aj keby ho predsla len istlo zmazat, tak to nie je dobry napad pretoze Y. Rovnako ako ked pre jeden object vypiseme viacero dovodov, preco ho nejde zmazat, hoci tieto dovody mozu vyplyvat jeden z druheho. Warningy su vlastne len nieco ako "slabsie" constrainy, tak nie je dovod ich osetrovat specialne.

Ci som si nieco neuvedomil a niekde tie warningy sposobia nieco nezmyselne/chybne?

Reply:
@martinmacko47 prepac ze som zmetkoval trocha. Definitivne, nechal som zobrazit constrainy a aj warningy.

Reply:
Este som zmenil XPath na `.//form//input[@type='submit']"`. Nepacilo sa mi ked po vyrendrovani  warningy a constrainy oddelovali zoznam `snoozed_actions`.

Reply:
Na druhe zamyslenie. Mozno by bloky s errormi/warningami bolo rozumnejsie presunut uplne hore, tesne pod nadpis. Nech je to prva vec, ktoru si admin vsimne.